templates/zerver/help/read-receipts.md

```diff
@@ -0,0 +1,83 @@
+# Read receipts
+
+Read receipts let you check who has read a message. You can see read receipts
+for any message, including both [stream messages](/help/streams-and-topics) and
+[private messages](/help/private-messages).
+
+With privacy in mind, Zulip lets you [control][configure-personal-read-recipts]
+whether your read receipts are shared, and administrators can
+[choose][configure-organization-read-recipts] whether to enable read receipts in
+their organization.
+
+!!! tip ""
+    Read receipts reflect whether or not someone has marked a message as read,
+    whether by viewing it, or [by marking messages as read in
+    bulk](/help/marking-messages-as-read).
+
+## View who has read a message
+
+{start_tabs}
+
+{!message-actions-menu.md!}
+
+3. Click **View read receipts**.
+
+!!! tip ""
+    You will also see how many people have read the message.
+
+{end_tabs}
+
+## Configure whether Zulip lets others see when you've read messages
+
+Zulip supports the privacy option of never sharing whether or not you have read
+a message.
```

Comment:
Maybe spell out explicitly that this will appear as though the user hasn't read any messages, implicitly, through them never appearing in the list of users who've read any message.

Reply:
Maybe add something like:
"If this setting is turned off, your name will never appear in the list of **Read receipts**."